.github/workflows/pr-tests-syft.yml

```diff
@@ -186,7 +186,7 @@ jobs:
           ORCHESTRA_DEPLOYMENT_TYPE: "${{ matrix.deployment-type }}"
           TEST_NOTEBOOK_PATHS: "${{ matrix.notebook-paths }}"
         with:
-          timeout_seconds: 1800
+          timeout_seconds: 2400
```

Comment:
Do we have problems with our tests?

Reply:
The tests timeout on model training, just trying if could resolve it.